config/groovy/common.groovy

```diff
@@ -215,8 +250,37 @@ class common {
                 println color("uncommitted changes. Skipping.", Ansi.YELLOW)
             } else {
                 println color("updating $itemType $itemName", Ansi.GREEN)
+                File targetDirFetchHead = new File("$targetDir/.git/FETCH_HEAD")
+                def timeLimit = use(groovy.time.TimeCategory){ 10.minute }
+                if (skipRecentUpdates && targetDirFetchHead.exists() && isRecentlyUpdated(targetDirFetchHead, timeLimit)){
+                    println color("Skipping update for $itemName: updated within last $timeLimit", Ansi.YELLOW)
+                    return
+                }
+                if (targetDirFetchHead.exists()){
+                    targetDirFetchHead.setLastModified(new Date().getTime())
+                }
+                
                 try {
+                    def current_sha = itemGit.log(maxCommits: 1).find().getAbbreviatedId(8)
                     itemGit.pull remote: defaultRemote
+                    def post_update_sha = itemGit.log(maxCommits: 1).find().getAbbreviatedId(8)
+                        
+                    if (current_sha != post_update_sha){
+                        println color("Updating $current_sha..$post_update_sha", Ansi.GREEN)
```

Comment:
Could you please add a `//TODO: ..` comment referencing the grgit issue

Reply:
done